@AH-64D Apache Longbow/Addons/fza_ah64_hellfire/functions/fn_arhSeeker.sqf

```diff
@@ -39,54 +40,56 @@ if !_isActive then {
     _seekerStateParams set [0, true];
 };
 
-if ((_lastTargetPollTime + (1 / 7)) - CBA_missionTime < 0) then {
-    _seekerStateParams set [4, CBA_missionTime];
-    private _searchPos = _calulatedSearchPos;
-    if (_searchPos isEqualTo [0, 0, 0]) exitwith {};
-    _target = objNull;
-    private _distanceToExpectedTarget = FCR_LIMIT_FORCE_LOBL_RANGE min ((getPosASL _projectile) vectorDistance _searchPos);
-
-    // Simulate how much the seeker can see at the ground
-    private _projDir = vectorDir _projectile;
-    private _projYaw = getDir _projectile;
-    private _rotatedYaw = (+(_projDir select 0) * sin _projYaw) + (+(_projDir select 1) * cos _projYaw);
-    if (_rotatedYaw isEqualTo 0) then { _rotatedYaw = 0.001 };
-    private _projPitch = atan ((_projDir select 2) / _rotatedYaw);
-    private _a1 = abs _projPitch;
-    private _a2 = 180 - ((_seekerAngle / 3) + _a1);
-    private _seekerBaseRadiusAtGround = ACTIVE_RADAR_MINIMUM_SCAN_AREA max (_distanceToExpectedTarget / sin(_a2) * sin(_seekerAngle / 2));
-    private _seekerBaseRadiusAdjusted = linearConversion [0, _seekerBaseRadiusAtGround, (CBA_missionTime - _lastTimeSeen) * vectorMagnitude _lastKnownVelocity, ACTIVE_RADAR_MINIMUM_SCAN_AREA, _seekerBaseRadiusAtGround, false];
-    if (_doesntHaveTarget) then {
-        _seekerBaseRadiusAdjusted = _seekerBaseRadiusAtGround;
-    };
-    // Look in front of seeker for any targets
-    private _nearestObjects = nearestObjects [ASLtoAGL _searchPos, ["all"], _seekerBaseRadiusAdjusted, false];
-    _nearestObjects = _nearestObjects select {([_projectile, [getpos _x, speed _x, _x], true] call fza_hellfire_fnc_limaLoblCheck)#1};
-    // Select closest object to the expected position to be the current radar target
-    if (_nearestObjects isEqualTo []) exitWith {
-        _projectile setMissileTarget objNull;
-        _expectedTargetPos
-    };
+if (([_projectile, [getpos _lastTarget, speed _lastTarget, _lastTarget], true] call fza_hellfire_fnc_limaLoblCheck)#1) then {
```

Comment:
Does this lead to the same issue we had with the original code where it would, when turning, latch onto the nearest-ish target to its path that it sees, not the correct one?